_posts/2021-08-29-mosaic-fusion.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Mosaic Image Fusion
+author: Volker Hilsenstein and Genevieve Buckley
+tags: [life science]
+theme: twitter
+---
+{% include JB/setup %}
+
+## Executive Summary
+
+This blogpost shows a case study, where a researcher uses Dask for mosaic image fusion - stitching multiple smaller images together seamlessly into a very large field of view. Full code examples are available on GitHub from the `DaskFusion` repository: https://github.com/VolkerH/DaskFusion
```

Comment:
This repository is currently private.

Reply:
I know, I was not sure whether the code would be included in the blogpost repo (in which case I would just included the relevant bits).

Reply:
I think it's probably more useful to link to a notebook or repository (that way people don't have to copy-paste bits together). 

Including small snippets of code in the blogpost might also be useful, if it's helpful to have an illustrative example?

Reply:
Note: The repository https://github.com/VolkerH/DaskFusion is public now